dpx/config.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "name": "Cognitive Control Predicts Use of Model-based Reinforcement learning",
+        "name": "Dot Pattern Expectancy,
         "run": [
```

Comment:
This line is missing a " after Expectancy so the json is valid. This would be caught in the validator, and I'll add an issue to the expfactory-python repo to make it possible to validate an experiment folder using the command line tool so you could more easily catch this.